test/Feature/ConcretizeSymbolicExternals.c

```diff
@@ -0,0 +1,35 @@
+// Check for calling external functions using symbolic parameters.
+// Externals calls might modify memory objects that have been previously fully symbolic.
+// The constant modifications should be propagated back.
+// RUN: %clang %s -emit-llvm -g -c -o %t.bc
+// RUN: rm -rf %t.klee-out
+// RUN: %klee --output-dir=%t.klee-out -external-calls=all %t.bc > %t.log
+// RUN: FileCheck -input-file=%t.log %s
+// REQUIRES: not-darwin
```

Comment:
Why not Darwin?

Reply:
Because the specific `printf` feature is not supported under Darwin.

Reply:
But then can we choose something more portable, maybe `sscanf`?

Reply:
Reminder about this